imageio/plugins/ffmpeg.py

```diff
@@ -140,6 +140,15 @@ class FfmpegFormat(Format):
         Values can be "quiet", "panic", "fatal", "error", "warning", "info"
         "verbose", or "debug". Also prints the FFMPEG command being used by
         imageio if not "quiet" or "warning".
+    macro_block_size: int
+        Size constraint for video width and height, must be divisible by this
+        number. If not divisible by this number imageio will tell ffmpeg to
```

Comment:
This first sentence seems a bit unclear. Do you mean "Size constraint for video. Width and height, must be divisible by this number."?